### DISCUSSION TITLE
Application title grayed out after launch

### DISCUSSION THREAD

**Discussion by mhalatek** [AvaloniaUI/Avalonia#13434] [Q&A] [answered]
After updating to Avalonia 11.0.4 I've stumbled upon a problem that my custom application title bar (title + window control buttons) is grayed out after launch. It activates only after clicking on buttons inside the app.

![image](https://github.com/AvaloniaUI/Avalonia/assets/130650323/c68bf446-2a53-47c9-9dec-1a4c170f79c4)
![image](https://github.com/AvaloniaUI/Avalonia/assets/130650323/d3bba992-6593-4f2d-8202-0a520518d788)
![image](https://github.com/AvaloniaUI/Avalonia/assets/130650323/71415c49-45f7-41b0-b184-22b382ad2d96)

With version 11.0.0-previev4 it worked fine - it was focused after launch. What should I do to make it work correctly?

**Comment by timunie** [comment]
check via DevTools F12 if Anything is focused at all. If not, call Focus() after loading.

**Comment by mhalatek** [comment, marked as answer]
Thank You for help. After checking DevTools I noticed nothing was focused after application launch. I added two lines in MainWindow.axaml.cs and now it works correctly.

public MainWindow()
        {
            InitializeComponent();
            Focusable = true;
            Focus();
        }
